art/step/step.py

```diff
@@ -54,11 +56,12 @@ def __call__(
             datamodule (L.LightningDataModule): Data module to be used.
```

Comment:
I just wonder why can't we create custom logger here (we will have it's id), after everything is runned we can just remove it by id. We could generate random log name from tempfile library and link it to the results so log path will be available from json. With this we also have logs for every run stored

Reply:
done with uuid